tests/schemas/api_v3/transactionsDryRun.schema.js

```diff
@@ -35,11 +35,21 @@ const event = {
 	id: Joi.string().pattern(regex.HEX).required(),
 };
 
+const eventSchemaWithSkipDecode = {
+	...event,
+	data: Joi.string().required(),
+};
+
 const dryrunTransactionSuccessResponseSchema = {
 	result: Joi.number().integer().valid(TRANSACTION_VERIFY_RESULT.OK).required(),
 	status: Joi.string().valid(...TRANSACTION_VERIFY_STATUSES).required(),
 	events: Joi.array().items(Joi.object(event).required()).min(1).required(),
 };
+const dryrunTxSuccessSchemaWithSkipDecode = {
+	result: Joi.number().integer().valid(TRANSACTION_VERIFY_RESULT.OK).required(),
+	status: Joi.string().valid(...TRANSACTION_VERIFY_STATUSES).required(),
```

Comment:
shouldn't success scenario have status as `OK`?

Reply:
Yes, it has status `OK`